examples/pytorch/rgcn/entity.py

```diff
@@ -0,0 +1,80 @@
+"""
+Differences compared to tkipf/relation-gcn
+* l2norm applied to all weights
+* remove nodes that won't be touched
+"""
+
+import argparse
+import torch as th
+import torch.nn.functional as F
+
+from torchmetrics.functional import accuracy
+
+from entity_utils import load_data
+from model import RGCN
+
+def main(args):
+    g, num_rels, num_classes, labels, train_idx, test_idx, target_idx = load_data(
+        args.dataset, get_norm=True)
+
+    num_nodes = g.num_nodes()
+
+    # Since the nodes are featureless, learn node embeddings from scratch
+    # This requires passing the node IDs to the model.
+    feats = th.arange(num_nodes)
```

Comment:
Maybe we shouldn't depend on this behavior anymore. How about we directly create an embedding tensor? Is it equivalent?

Reply:
Mathematically, it is not equivalent primarily due to the basis-decomposition in initializing node embeddings. Meanwhile, it's possible to initialize a learnable embedding per node without basis decomposition as in the implementation of sampling-based entity classification. However, it will require additional efforts to tune the hyperparameters to see if we can match the performance numbers.